### DISCUSSION TITLE
Add a way to reset one goal only

### DISCUSSION THREAD

**Discussion by divinerites** [plausible/analytics#310] [Anything Else]
## Prerequisites
- [X] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request

When you are developing & setting a goal on your code, you need to check if it works on plausible prod.

When you're facing difficult debug (or you're just a bad coder like me) your goal already will have some decent number before going in production.
So you want to reset it.
But at the moment, removing the goal from the interface, keep the analytic informations for this goal.

**How would you like it to work**

It would be helpfull for those dev/debug cases to be able to "really" reset stats for a specific goal.

**Comment by jis3r** [comment]
I would defenitely need a feature like this too!
